### DISCUSSION TITLE
[提问]萌新的角色培养问题

### DISCUSSION THREAD

**Discussion by Zhenjun3** [share121/inter-knot#3084] [其他]
现在建不建议我抽_简_，应该怎么_配队_？**_异常属性_**搞不明白，我看到好看的角色就想抽😶
![1724057918469.jpg](https://github.com/user-attachments/assets/de8f2952-25d6-4e2b-aada-512f7cda8898)

**Comment by yitan1741** [comment]
建议抹茶小蛋糕（可琳）c哦，正好练青衣了，辅助随便带一个，二队就成了，伤害还是很可观的，建议先不要练格莉丝了，紊乱队手法要求比较高。

至于异常属性，异常精通是让你右边圆条打满后的伤害增加，异常掌控是更快让你打满右边的圆条。至于条打满后的效果，建议前往官网查询，口述较为复杂，且我的理解也非常片面<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3996afc68a13bfc5b5718af31e610f43_6807527097924316704.png" alt="罗刹 送花" width="75" >

**Reply by yitan1741** [reply]
至于抽不抽简，喜欢就抽喽，不喜欢就算了，游戏嘛，怎么开心怎么来

**Reply by Zhenjun3** [reply]
谢谢啦੭ ᐕ)੭*⁾⁾
